packages/api-browser/window.js

```diff
@@ -2,10 +2,20 @@ if (!window.ssf) {
   window.ssf = {};
 }
 
+window.accessableWindows = [];
```

Comment:
> If possible, I think we should avoid polluting the global namespace like this. Perhaps, for now, `accessibleWindows` could be a `const` and there could be `static` methods on `Window` to allow this to be get/set?

For this - I get that this would be easier to do this if this code is in the same file or when the ES6 modules are implemented - so let me know if you'd like to discuss further :)